pytest_html/plugin.py

```diff
@@ -213,9 +213,15 @@ def append_extra_html(self, extra, extra_index, test_index):
         def append_log_html(self, report, additional_html):
             log = html.div(class_='log')
             if report.longrepr:
-                for line in str(report.longrepr).splitlines():
+                if hasattr(report, 'longreprtext'):
```

Comment:
I believe `report.longreprtext` should already be a `unicode` string at this point: the recommended way to deal with unicode is to usually encode/decode at the boundaries; otherwise all clients must be aware how to decode it from `bytes`.